lib/backend/model/ippool.go

```diff
@@ -27,15 +27,15 @@ import (
 )
 
 var (
-	matchPool = regexp.MustCompile("^/?calico/v1/ipam/v./pool/([^/]+)$")
-	typePool  = reflect.TypeOf(Pool{})
+	matchIPPool = regexp.MustCompile("^/?calico/v1/ipam/v./pool/([^/]+)$")
```

Comment:
nope, the etcdv2 key is a "stable" api itself, and has to be compatible with the python version (which uses `pool`)